composer.json

```
@@ -16,7 +16,7 @@
     "require": {
         "php": "^7.4 | ^8.0",
         "consolidation/robo": "3.0.*",
```

```suggestion
        "consolidation/robo": "3.0.* | 4.0.*",
```
For symfony 6 you need also robo 4.0.* :)